test/lib/scsi/scsi.sh

```diff
@@ -12,5 +12,6 @@ $valgrind $testdir/dev/dev_ut
 $testdir/init/init_ut
 $valgrind $testdir/lun/lun_ut
 $testdir/scsi_bdev/scsi_bdev_ut
+$testdir/scsi_nvme/scsi_nvme_ut
```

Comment:
This test should be able to run under Valgrind since it does not use DPDK (add $valgrind to the beginning of the line like the lun_ut above).